# Question 32
-----

In [1]:
import math

occ_effect = math.exp(-0.5309)
print(f"OCC effect: {occ_effect}")

OCC effect: 0.588075463519158


In [2]:
1 / 0.588

1.7006802721088436

Hence, a person with $OCC = 0$ is 1.7 times more likely to develop hypertension than a person with $OCC = 0$